pkg/timeutils/utils.go

```diff
@@ -9,12 +9,8 @@ import (
 // or Unix timestamp, if successful returns a Unix timestamp
 // as string otherwise returns value back.
 func GetTimestamp(value string) string {
-	format := RFC3339NanoFixed
 	loc := time.FixedZone(time.Now().Zone())
-	if len(value) < len(format) {
```

Comment:
I think it was added, and related, to this PR/comment:  https://github.com/docker/docker/pull/2348#issuecomment-27168600